docs/content/latest/yugabyte-platform/configure-yugabyte-platform/set-up-cloud-provider/openshift.md

```diff
@@ -0,0 +1,210 @@
+```
+
+The next step is to grant access to this ServiceAccount using Roles and RoleBindings, thus allowing it to manage the YugabyteDB universe's resources for you. To create the required RBAC objects, execute the following command:
+
+```shell
+curl -s https://raw.githubusercontent.com/yugabyte/charts/master/rbac/platform-namespaced.yaml \
+ | sed "s/namespace: <SA_NAMESPACE>/namespace: yb-platform/g" \
+ | oc apply -n yb-platform -f -
+# output
+role.rbac.authorization.k8s.io/yugabyte-helm-operations created
+role.rbac.authorization.k8s.io/yugabyte-management created
+rolebinding.rbac.authorization.k8s.io/yugabyte-helm-operations created
+rolebinding.rbac.authorization.k8s.io/yugabyte-management created
+```
+
+The next step is to create a kubeconfig for this ServiceAccount. You download a helper script for generating a kubeconfig file by executing the following command:
+
+```shell
+wget https://raw.githubusercontent.com/YugaByte/charts/master/stable/yugabyte/generate_kubeconfig.py
+```
+
+To generate the kubeconfig file, execute the following:
+
+```shell
+python generate_kubeconfig.py \
+ --service_account yugabyte-platform-universe-management \
+ --namespace yb-platform
+# output
+Generated the kubeconfig file: /tmp/yugabyte-platform-universe-management.conf
+```
+
+## Creating a Provider in Yugabyte Platform
+
+Since Yugabyte Platform manages YugabyteDB universes, Yugabyte Platform needs details about the cloud providers. In your case, the provider is your own OCP cluster.
+
+You can create a provider as follows:
+
+- Open the Yugabyte Platform web UI and click **Configure a Provider** to open the **Cloud Provider Configuration** page shown in the following illustration.
+- Select **Red Hat OpenShift** and complete the fields, as follows:
+  - In the **Type** filed, select **OpenShift**.
+  - In the **Name** field, enter ocp-test.
+  - Use the **Kube Config** field to select the file that you created in the preceding step.
+  - In the **Service Account** field, enter yugabyte-platform-universe-management.
+  - In the **Image Registry** field, if you are performing Operator-based installation, use  `registry.connect.redhat.com/yugabytedb/yugabyte`, and if you are performing Helm-based installation, use  `quay.io/yugabyte/yugabyte-ubi`
+- Optionally, use the **Pull Secret File** field to upload the pull secret you received from Yugabyte Support. 
+
+![OpenShift Provider Config](/images/ee/openshift-cloud-provider-setup.png)
+
+- Click **Add Region** and complete the **Add new region** dialog shown in the following illustration by first selecting the region you found previously (US East), and then entering the following information:
+  - In the **Zone** field, enter the exact zone label (us-east4-a).
+  - In the **Namespace** field, enter yb-platform.
+
+![img](https://lh5.googleusercontent.com/gQ-fTRZnDTp4kCqNaKo8KUgVA2mJeCCjaHiqzdCOcG4350yxgDGZojMWhsfdvcpLJbOio8sL8K932wmDM6_S8fIkL9wfKAo-b3340Yvc1Dy4FJ61o2Ec93DptkB_Ski1XqaU3UrT)
```

Comment:
Please use ocp-yugabyte-platform-provider-2.png from Drive.

Reply:
I, actually, saved all the images on the drive already and replaced them in the doc.

Reply:
Those needs to be added here, the Google links we are using here are not permanent, they might go away, or change later (that's what I have seen in past).

Reply:
Yes, I did save them on the site. They will be visible when I update the PR.